redeal/redeal.py

```diff
@@ -256,31 +265,37 @@ def prepare(cls, predeal):
             dealer["_smartstack"] = seat
         dealer["_remaining"] = [card for card in FULL_DECK
                                 if card not in predealt_set]
-        return lambda: cls(dealer)
+        return lambda accept_func=None, tries=1000: cls(dealer, accept_func, tries)
```

Comment:
would be easier to write `functools.partial(cls, dealer)` at that point.

Reply:
Absolutely! Didn't think about it. I've read about `functools.partial` in Fluent Python, but haven't used it myself when programming. Nice to find a case for it :)